daq/faucet_states_collector.py

```diff
@@ -108,6 +111,82 @@ def get_switch(self, switch_name):
 
         return switch_map
 
+    def get_active_host_route(self, src_mac, dst_mac):
+        """give two MAC addresses in the core network, find the active route between them"""
+        def get_switches_ports_from_link(link_map):
+            return (link_map["port_map"]["dp_a"], int(link_map["port_map"]["port_a"][5:])), \
```

Comment:
find a way to remove line break

 extract link_map["port_map"] to local variable?

Reply:
Done